src/utils/Theme/themes/dark.ts

```diff
@@ -411,6 +411,16 @@ export const darkTheme: Theme = {
     },
     calendar: {
       'calendar-spacing': '8px'
+    },
+    badge: {
+      'badge-color-default': darkColors.white,
+      'badge-color-on-default': darkColors.black,
```

Comment:
We don't use this `on` pattern in the other spots, let's match that style please.

Reply:
Updated but I saw that pattern at [another place](https://github.com/reaviz/reablocks/blob/4f8943f22512564be66ce8836d7e3ba11e04d7f0/src/utils/Theme/themes/dark.ts#L219-L220) as well.